### PR TITLE
Add facts about seahorse #298

### PR DESCRIPTION
**Pull request by wigsnes**

### CODE REVIEW

**Comment by wigsnes** [comment]
#298

File: animalfacts.py [hunk 1]
```diff
@@ -2453,6 +2454,25 @@ def animalfactsbot(reddit):
...
+    'The seahorse feeds constantly on plankton and tiny fish. It moves each of its eyes independently, so it can follow the activity of passing sea life without giving its presence away.',
+    'Seahorses have no teeth and no stomach. Food passes through their digestive systems so quickly, they must eat almost constantly to stay alive.',
+    'They can consume 3,000 or more brine shrimp per day.',
+    'Seahorses are monogamous and mate for life.',
+    'Rarer still, they are among the only animal species on Earth in which the male bears the unborn young.',
+    'Male pregnancy frees to female to make more eggs straight away and so reproduce quicker.',
+    'Seahorses engage in an eight hour courtship dance which includes spinning around, swimming side by side and changing colours.',
+    'When mating, the female seahorse releases up to 50 eggs into a pouch on the male’s abdomen.',
+    'The male seahorse carries the eggs in his pouch until they hatch, then releases fully formed, miniature seahorses into the water.  As little as 5 or as many as 1,500 young can be born.'
```
Comment (mpentler) [inline animalfacts.py]: Could you put a comma at the end here? Just to match the other tuples?

Comment (wigsnes) [reply animalfacts.py]: a, shoot. how did I miss that.

File: animalfacts.py [hunk 2]
```diff
@@ -2453,6 +2454,25 @@ def animalfactsbot(reddit):
...
+    'The seahorse feeds constantly on plankton and tiny fish. It moves each of its eyes independently, so it can follow the activity of passing sea life without giving its presence away.',
+    'Seahorses have no teeth and no stomach. Food passes through their digestive systems so quickly, they must eat almost constantly to stay alive.',
+    'They can consume 3,000 or more brine shrimp per day.',
+    'Seahorses are monogamous and mate for life.',
+    'Rarer still, they are among the only animal species on Earth in which the male bears the unborn young.',
+    'Male pregnancy frees to female to make more eggs straight away and so reproduce quicker.',
+    'Seahorses engage in an eight hour courtship dance which includes spinning around, swimming side by side and changing colours.',
+    'When mating, the female seahorse releases up to 50 eggs into a pouch on the male’s abdomen.',
+    'The male seahorse carries the eggs in his pouch until they hatch, then releases fully formed, miniature seahorses into the water.  As little as 5 or as many as 1,500 young can be born.',
+)
```
Comment (mpentler) [inline animalfacts.py]: Can I be a huge bastard and ask that you indent this line as well? My OCD is a living nightmare.

**Comment by wigsnes** [comment]
DONE

**Pull request merged**